app/src/pages/common/settingsPage/notificationsTab/notificationCaseList/caseListItem/caseListItem.jsx

```diff
@@ -36,7 +36,10 @@ const ruleFieldsConfig = {
   [ATTRIBUTES_FIELD_KEY]: {
     title: messages.attributesLabel,
     dataFormatter: (data) =>
-      data.reduce((acc, item) => `${acc.length ? `${acc}, ` : ''}${item.key}: ${item.value}`, ''),
+      data.reduce(
+        (acc, { key, value }) => `${acc.length ? `${acc}, ` : ''}${key ? `${key}:` : ''}${value}`,
```

Comment:
Can `formatAttribute` from `attributeUtils.js` be used here?

Reply:
Yes. Thanks. Fixed